docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,341 @@
+.. _install-server-with-podman:
+
+Install Inmanta with Podman and Systemd
+***************************************
+
+This page explains how to setup an Inmanta orchestration server using Podman and Systemd.
+This guide assumes you already have `Podman <http://podman.io/>`_ installed on your machine and that you are running a Linux distribution with Systemd.
+
+.. note::
+    The full setup should be doable without any root privilege (rootless) on the host, running the orchestrator with an unprivileged user.  
```

Comment:
```suggestion
    The full setup can be done root privileges (rootless) on the host and running the orchestrator with an unprivileged user.  
```

Reply:
I think a word is missing there, I am not sure of the point you are making

Reply:
Like this. What I changed was the your version is very conditional: "should be doable" sounds like: we have not tried it but we so no reason why it would not work

Reply:
I formulated it completely differently, can you check again?